data_pipeline/generic_web_api/url_builder.py

```diff
@@ -51,6 +51,7 @@ def __init__(
             compose_able_url_key_val: dict = None,
             sort_key: str = None,
             sort_key_val: str = None,
+            default_start_end_date_diff_in_days: int = None,
```

Comment:
`default_start_end_date_diff_in_days` doesn't seem to be used in this class.

Reply:
fixed